ArduCopter/GCS_Mavlink.cpp

```diff
@@ -772,7 +772,8 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_long_packet(const mavlink_command_
 
 #if MODE_AUTO_ENABLED == ENABLED
     case MAV_CMD_DO_LAND_START:
-        if (copter.mode_auto.jump_to_landing_sequence_auto_RTL(ModeReason::GCS_COMMAND)) {
+        copter.set_mode_auto_do_land_start_or_RTL(ModeReason::GCS_COMMAND);
```

Comment:
This is a little odd because it defualts to the other modes, that is a little un-expected via mavlink command.